src/components/button/toggling-button.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Create Toggling Button
+ * Adds an event listener to toggle the `aria-pressed` value
+ *
+ * @param togglingButton - The button to apply the toggling functionality to
+ */
+export const initTogglingButton = (togglingButton: HTMLButtonElement) => {
+  // Handler for when a toggling button is clicked
+  const onTogglingButtonClick = () => {
+    // Get the current aria-pressed state and toggle it
+    const isAriaPressed = !(
+      togglingButton.getAttribute('aria-pressed') === 'true'
+    );
```

Comment:
```suggestion
    const isAriaPressed =
      togglingButton.getAttribute('aria-pressed') !== 'true';
```

Reply:
Thanks for calling this section out, I wasn't happy with it and found it confusing. I like your suggestion but also it still feels a bit confusing to me.

I decided to refactor it, do you feel this is more straightforward? https://github.com/cloudfour/cloudfour.com-patterns/pull/1318/commits/7a73ff30441c4bc16e8cdc06f822e4e3dfa76085

Reply:
Yes that makes more sense 👍